profiles-jar-examples/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Mon Jun 06 03:35:26 CDT 2022
+gradle.version=7.4
```

Comment:
Need to remove the `.gradle/` dir - guess it was added before the .gitignore took place ? (I can include this when porting over my changes)

Reply:
yeah..oops